docs/migration-recipe.md

```diff
@@ -0,0 +1,117 @@
+Migrating to new host:
+----------------------
+
+### Prepare salt configuration for migration:  
+
+1.  Ensure that salt-master, loadbalancer, and host in question are up, and that their health check for the relevant service is failing in haproxy after the host is fully up
+
+To view haproxy status: 
+
+-vagrant up the salt-master, loadbalancer, and host in question (`vagrant up`)
```

Comment:
if you correct the markdown formatting to have a space after the leading `-` here and on the next few lines you should get a pretty list!